### PR TITLE
docs: update docs with better visuals and guides

### PR DESCRIPTION
**Pull request by luizstacio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,48 @@
...
+  - [📗 Project overview](docs/GETTING_STARTED.md#-project-overview)
+  - [🧰 Useful scripts](docs/GETTING_STARTED.md#-useful-scripts)
+- [Contribution Guide](docs/CONTRIBUTE_GUIDE.md)
+  - [Finding Something to Work On](docs/CONTRIBUTE_GUIDE.md#finding-something-to-work-on)
+  - [Contribution Flow](docs/CONTRIBUTE_GUIDE.md#contribution-flow)
+- [License](#license)
+
+## 🧰 SwaySwap Features
+
+- Faucet coins; Use the faucet API to send some EThs to your wallet
```
Comment (matt-user) [inline README.md:33]: I think this should be "ETH" and not "EThs"

File: README.md [hunk 2]
```diff
@@ -1,33 +1,48 @@
...
+- Mint tokens; Use the Token Contract to mint some EThs to your wallet
```
Comment (matt-user) [inline README.md:34]: Are we minting tokens or using a faucet for them?  Also we are minting/faucet the tokens not EThs right?

File: docs/GETTING_STARTED.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+Start the local development `front-end`. After running the command you can open on
+you browser [http://localhost:3000](http://localhost:3000).
+
+```
+pnpm dev
+```
+
+### ✨ - First steps
+
+This is guided step-by-step tutorial to create your first pool and be able to
```
Comment (matt-user) [inline docs/GETTING_STARTED.md:56]: I think the correct grammar is "This guided step-by-step tutorial will allow you to create your first pool and play around with the application"

**Review by matt-user — CHANGES_REQUESTED** [review]
I think overall the changes provide useful information to new users looking at the repository.  I just noticed some small grammatical errors.

File: README.md (hunk 2 above)
Comment (luizstacio) [reply README.md:34]: we do both, the faucet is to ETH and mint to DAI, I wrote the wrong name haha thanks

**Review by pedronauck — APPROVED** [review]

**Pull request merged**